### PR TITLE
Minor UI patch

### PR DESCRIPTION
**Pull request by ejcheng**

### CODE REVIEW

File: src/modules/nav/index.tsx [hunk 1]
```diff
@@ -98,7 +99,19 @@ export const Navbar = () => {
...
+              cursor: 'pointer',
+              }}
+              >
+              <GitHub style={{
+                color: 'white',
+                borderRadius: '50px',
+                padding: '8px',
+              }} size="3rem" />
+            </Button>
+          </NextLink>
```
Comment (mikayelgr) [inline src/modules/nav/index.tsx:114]: Not too sure about this one to be honest... There is already a GitHub button in the footer section redirecting to our GitHub page.

Comment (ejcheng) [reply src/modules/nav/index.tsx:114]: I mean, I think a lot of people wouldn't scroll all the way down to the footer. It's personal preference though

Comment (mikayelgr) [reply src/modules/nav/index.tsx:114]: I see. In that case, I think it would be nice to set `spacing={1}` for the parent `Stack` component.

Comment (ejcheng) [reply src/modules/nav/index.tsx:114]: Done

**Review by mikayelgr — APPROVED** [review]
LGTM!

**Comment by mikayelgr** [comment]
Thanks for your contribution!

**Pull request merged**